app/models/smart_proxy.rb

```diff
@@ -180,6 +188,6 @@ def get_features
   end
 
   class Jail < ::Safemode::Jail
-    allow :id, :name, :hostname, :httpboot_http_port, :httpboot_https_port
+    allow :id, :name, :hostname, :httpboot_http_port, :httpboot_https_port, :httpboot_http_port!, :httpboot_https_port!
```

Comment:
@ares suggested we don't want to deal with exceptions in templates. Do we need those variants in the jail?

Reply:
it may be still handy for people, who prefer to fail instead of skip the menuentry, let's keep it in for now, I'll also open a ticket to document SmartProxy Jail in templates DSL, we don't have it yet